gemcalibration/src/common/CalibrationWeb.cc

```diff
@@ -0,0 +1,394 @@
+#include "gem/calib/CalibrationWeb.h"
+#include <boost/algorithm/string.hpp>
+#include "xcept/tools.h"
+#include "gem/base/utils/GEMInfoSpaceToolBox.h"
+#include "gem/utils/exception/Exception.h"
+#include "gem/calib/Calibration.h" 
+#include <iomanip>
+
+gem::calib::CalibrationWeb::CalibrationWeb(gem::calib::Calibration* CalibrationApp) :
+  gem::base::GEMWebApplication(CalibrationApp)
+{
+    // default constructor
+}
+
+gem::calib::CalibrationWeb::~CalibrationWeb()
+{
+  // default destructor
+}
+
+void gem::calib::CalibrationWeb::webDefault(xgi::Input * in, xgi::Output * out)
+  throw (xgi::exception::Exception)
+{
+  *out << cgicc::script().set("type", "text/javascript")
+    .set("src", "/gemdaq/gemcalibration/html/scripts/cal_routines.js") 
+       << cgicc::script() << std::endl;
+  
+  *out << "<link  rel=\"stylesheet\" href=\"/gemdaq/gembase/html/css/bootstrap.css\" type=\"text/css\">" << std::endl;
+  *out << "<link  rel=\"stylesheet\" href=\"/gemdaq/gembase/html/css/bootstrap.min.css\" type=\"text/css\">" << std::endl;
+  *out << "<script src=\"/gemdaq/gembase/html/scripts/bootstrap.js\"></script>" << std::endl;
+  *out << "<script src=\"https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js\"></script>" << std::endl;
+
+  *out << "<div class=\"xdaq-tab-wrapper\">" << std::endl;
```

Comment:
Formatting in this file *must* be for `c++`, not `html`.
If you also want formatted `html` in the output, ensure what you write to the stream is formatted as desired.